kubernetes/ansible/roles/secor-deploy/defaults/main.yaml

```diff
@@ -5,6 +5,10 @@ image_pullPolicy: IfNotPresent
 imagepullsecrets: sunbird-registry-secret
 secor_storage_class: managed-premium
 
+# this varible is to control the monitoring deployment. override this value by adding into
+# private repo common.yml to not to create the secor lag alert rules.
+secor_alertrule_enabled: true
+core_kubeconfig_path: "" # This var being used to create the secor lag alert rule in core prometheus.
```

Comment:
Sorry but I didn't get why are we doing this ?

Reply:
When adopters deploying the secor jobs and if they don't have a monitoring stack installed then the deployment job will fail. To avoid that we introduced this var. 

I kept default as true to let the adopter override it. Do we need to make it false by default? @rjshrjndrn

Reply:
My doubt is more on the variable part, why do you need the kubeconfig path? if the monitoring stack is not there, can't we just skip the entire process? using helm or something?

Reply:
we are using the var in ansible-playbook in the export command. Thought if we don't define the variable then ansible-playbook will fail. Hence added that. 

Now removed the var and added the condition in jenkinsfile to skip the execution of the playbook. @rjshrjndrn